### ISSUE TITLE
🪲  levels->3->1->mp_choice_options->[3]->option chinese version not work

### ISSUE THREAD

**Issue by ZoomQuiet** [hedyorg/hedy#5566]
**Describe the bug**
can not pass quiz

**Paste the Hedy code & level**
base translation:
levels->3->1->mp_choice_options->[3]->option

**Add a screenshot (optional)**
![50831716798521_ pic](https://github.com/hedyorg/hedy/assets/22494/872579ff-db64-4daa-af59-47cb32418bc2)

- try quiz in level 3
- 1st question
- whatever can not pass, click any one will got:"No such question"

**What machine and browser you were using (optional)**
If the issue concerns things in the website, let us know:

- in macOS 14.5 (M1 Max)
- Chrome v125.0.6422.77

**Comment by hasan-sh** [comment]
@ZoomQuiet could you do a hard refresh and retry? Because i couldn't reproduce the error! Or make a screencast to follow the exact steps you took when you encountered this error?

**Comment by ZoomQuiet** [comment]
> @ZoomQuiet could you do a hard refresh and retry? Because i couldn't reproduce the error! Or make a screencast to follow the exact steps you took when you encountered this error?

such as this video:

https://github.com/hedyorg/hedy/assets/22494/993cc1b3-5197-4423-93c4-e7bc04b8bff7

there 3 class:
- dama240501: https://hedy.org/for-teachers/class/bf819bbccf66454b8ca9eee1cf822f63
    + created myself, students by create
    + can not enter `live_stats/class`, ERROR:`This class does not exist`
- 101camp : https://hedy.org/for-teachers/class/7c69cc04f0974c6b965c7b08940fd303
    + created myself, students by invite link
    + can enter `live_stats/class`, but click student name's link ERROR:`我们无法找到该页面... None`
- 留学生2024春: https://hedy.org/for-teachers/class/96f00447017f424fb877ade2fc21d045
    + created by friend, Chaos42dama as teacher and students all by invite link enter
    + can not enter `live_stats/class`, ERROR:`This class does not exist`

and after i switch back english interface, same error;

so, i can not know, what happened:

- why the default functions can not usage now?

**Comment by hasan-sh** [comment]
@ZoomQuiet thanks for making this screencast:)) this is a different error than the one you described in this issue. If you have any more info about the original issue (i.e., the quiz!) please feel free to share so that I can find out what the problem is!


> such as this video:

For this video, I'll ask @jpelay to take a look as well and hopefully we will find out what the problem is!

**Comment by jpelay** [comment]
> > @ZoomQuiet could you do a hard refresh and retry? Because i couldn't reproduce the error! Or make a screencast to follow the exact steps you took when you encountered this error?
> 
> such as this video:
> 
>  240529hedy-class-bug.mp4 
> there 3 class:
> 
> * dama240501: https://hedy.org/for-teachers/class/bf819bbccf66454b8ca9eee1cf822f63
>   
>   * created myself, students by create
>   * can not enter `live_stats/class`, ERROR:`This class does not exist`
> * 101camp : https://hedy.org/for-teachers/class/7c69cc04f0974c6b965c7b08940fd303
>   
>   * created myself, students by invite link
>   * can enter `live_stats/class`, but click student name's link ERROR:`我们无法找到该页面... None`
> * 留学生2024春: https://hedy.org/for-teachers/class/96f00447017f424fb877ade2fc21d045
>   
>   * created by friend, Chaos42dama as teacher and students all by invite link enter
>   * can not enter `live_stats/class`, ERROR:`This class does not exist`
> 
> and after i switch back english interface, same error;
> 
> so, i can not know, what happened:
> 
> * why the default functions can not usage now?

Hi, @ZoomQuiet! Thanks for reporting the error! Right now we are focusing on phasing out that page, to offer a more useful view of statistics to teachers, we are expecting it to be ready within the next few days.

**Comment by boryanagoncharenko** [comment]
About the quiz error, I noticed that something is off with the quotes. When the translated string contains two consecutive backticks, they seem to be displayed, which is incorrect. For example, if I set levels->3->1->mp_choice_options->[3]->option to ``` `{at}``{random}` ``` (without having the space between the keywords), the quiz displays ```at``random``` instead of `at random`.

In the case of Chinese, this should produce the output ``` 在``随机 ```. However, the screenshot shows ``` 在''随机 ```. So why are there single quotes instead of backticks? @ZoomQuiet do you have a clue?

**Comment by ZoomQuiet** [comment]
> About the quiz error, I noticed that something is off with the quotes. When the translated string contains two consecutive backticks, they seem to be displayed, which is incorrect. For example, if I set levels->3->1->mp_choice_options->[3]->option to `` `{at}``{random}` `` (without having the space between the keywords), the quiz displays `` at``random `` instead of `at random`.
> 
> In the case of Chinese, this should produce the output `` 在``随机 ``. However, the screenshot shows `在''随机`. So why are there single quotes instead of backticks? @ZoomQuiet do you have a clue?

i need check https://hosted.weblate.org/projects/hedy/-/zh_Hans/ some item,
i guess is error in translation words....

**Comment by boryanagoncharenko** [comment]
> > About the quiz error, I noticed that something is off with the quotes. When the translated string contains two consecutive backticks, they seem to be displayed, which is incorrect. For example, if I set levels->3->1->mp_choice_options->[3]->option to ``  `{at} ``{random}`` `` (without having the space between the keywords), the quiz displays `` at``random `` instead of ``at random`.
> > In the case of Chinese, this should produce the output ` 在`随机 ``. However, the screenshot shows `在''随机`. So why are there single quotes instead of backticks? @ZoomQuiet do you have a clue?
> 
> i need check https://hosted.weblate.org/projects/hedy/-/zh_Hans/ some item, i guess is error in translation words....

I replaced all occurrences of double backticks including the one for simplified Chinese. It should solve the appearance of the quiz question. However, I doubt that it will address the `Bad request No such question` error.

Are you still able to replicate the bad request error on main or on hedy.org? If so, are you logged in and do you have a preferred keyword language set? Thanks in advance for your input!

**Comment by boryanagoncharenko** [comment]
Reopening the issue until we have a conclusion about the Bad request error

**Comment by Felienne** [comment]
I though in the meeting @hasan-sh said this cannot be reproduced @boryanagoncharenko. So I think we can close it? (if I understood correctly, feel free to close)!

**Comment by boryanagoncharenko** [comment]
Yes, we cannot replicate the error. I noticed a small issue with quoting and addressed it, but I have not clue how to reproduce the single quotes in the middle of `at random` or the 'No such question' error. The latter problem can occur only if the question loaded once correctly (because we see the question rendered on the screen) but failed to load for the second time when we are trying to submit an answer.

Closing the issue for now. @ZoomQuiet if you happen to replicate the problem again, do not hesitate to reopen it.
